recipes/django/meta.yaml

```diff
@@ -0,0 +1,230 @@
+        #- django.contrib.admin.views
+        - django.contrib.admindocs
+        - django.contrib.admindocs.tests
+        #- django.contrib.auth
+        #- django.contrib.auth.handlers
+        #- django.contrib.auth.management
+        #- django.contrib.auth.management.commands
+        #- django.contrib.auth.migrations
+        #- django.contrib.auth.tests
+        - django.contrib.contenttypes
+        - django.contrib.contenttypes.migrations
+        - django.contrib.flatpages
+        - django.contrib.flatpages.migrations
+        - django.contrib.flatpages.templatetags
+        - django.contrib.gis
+        #- django.contrib.gis.admin
+        - django.contrib.gis.db
+        - django.contrib.gis.db.backends
+        - django.contrib.gis.db.backends.base
+        - django.contrib.gis.db.backends.mysql
+        - django.contrib.gis.db.backends.oracle
+        - django.contrib.gis.db.backends.postgis
+        - django.contrib.gis.db.backends.spatialite
+        #- django.contrib.gis.db.models
+        #- django.contrib.gis.db.models.sql
+        - django.contrib.gis.forms
+        - django.contrib.gis.gdal
+        - django.contrib.gis.gdal.prototypes
+        - django.contrib.gis.gdal.raster
+        #- django.contrib.gis.geoip
+        - django.contrib.gis.geometry
+        #- django.contrib.gis.geometry.backend
+        - django.contrib.gis.geos
+        - django.contrib.gis.geos.prototypes
+        - django.contrib.gis.management
+        - django.contrib.gis.management.commands
+        - django.contrib.gis.maps
+        - django.contrib.gis.maps.google
+        - django.contrib.gis.maps.openlayers
+        - django.contrib.gis.serializers
+        #- django.contrib.gis.sitemaps
+        - django.contrib.gis.utils
+        - django.contrib.humanize
+        - django.contrib.humanize.templatetags
+        - django.contrib.messages
+        - django.contrib.messages.storage
+        - django.contrib.postgres
+        #- django.contrib.postgres.fields
+        - django.contrib.postgres.forms
+        - django.contrib.redirects
+        - django.contrib.redirects.migrations
+        - django.contrib.sessions
+        - django.contrib.sessions.backends
+        - django.contrib.sessions.management
+        - django.contrib.sessions.management.commands
+        - django.contrib.sessions.migrations
+        - django.contrib.sitemaps
+        - django.contrib.sitemaps.management
+        - django.contrib.sitemaps.management.commands
+        - django.contrib.sites
+        - django.contrib.sites.migrations
+        - django.contrib.staticfiles
+        - django.contrib.staticfiles.management
+        - django.contrib.staticfiles.management.commands
+        - django.contrib.staticfiles.templatetags
+        - django.contrib.syndication
+        - django.contrib.webdesign
+        - django.contrib.webdesign.templatetags
+        - django.core
+        #- django.core.cache
+        #- django.core.cache.backends
```

Comment:
Importing these packages won't work outside of a Django context (app settings are defined).

```
>>> import django.core.cache
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/kwilcox/miniconda3/envs/platforms/lib/python3.4/site-packages/django/core/cache/__init__.py", line 34, in <module>
    if DEFAULT_CACHE_ALIAS not in settings.CACHES:
  File "/home/kwilcox/miniconda3/envs/platforms/lib/python3.4/site-packages/django/conf/__init__.py", line 48, in __getattr__
    self._setup(name)
  File "/home/kwilcox/miniconda3/envs/platforms/lib/python3.4/site-packages/django/conf/__init__.py", line 42, in _setup
    % (desc, ENVIRONMENT_VARIABLE))
django.core.exceptions.ImproperlyConfigured: Requested setting CACHES, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings.
```